.circleci/config.yml

```diff
@@ -25,12 +33,129 @@ jobs:
       - run:
           name: Root package install
           command: npm ci
+      - save_cache:
+          key: deps-{{ checksum "package-lock.json" }}
+          paths:
+            - ./node_modules
+
+  lint:
+    working_directory: ~/stockflux
+    docker:
+      - image: circleci/node:10.15.1
+    steps:
+      - checkout:
+          path: ~/stockflux
+      - restore_cache:
+          key: deps-{{ checksum "package-lock.json" }}
       - run:
-          name: Bootstrap packages
+          name: Lint *.js files
+          command: npm run lint
+
+  build:
+    working_directory: ~/stockflux
+    docker:
+      - image: circleci/node:10.15.1
+    steps:
+      - checkout:
+          path: ~/stockflux
+      - restore_cache:
+          key: deps-{{ checksum "package-lock.json" }}
+      - run:
+          name: Restore Bitflux Cache
+          command: cd ~/stockflux/packages/stockflux-bitflux
```

Comment:
I'm pretty sure that the `run` step runs a new shell each time and is rooted at the `working_directory` by default, each time it is executed. This means that this command which changes to the `~/stockflux/packages/stockflux-bitflux` directory effectively does nothing. The following step of restoring the cache with the key `deps-bitflux-{{ checksum "package-lock.json" }} is actually using the `~/stockflux/package-lock.json` as the source file for the checksum, which isn't what we want.